groups/instance/variables.tf

```diff
@@ -4,12 +4,24 @@
 
 variable "account_name" {
   description = "The name of the AWS account we are using"
+  default     = "shared-services"
+  type        = string
+}
+
+variable "secrets_account_name" {
+  description = "The name of the AWS account to be used when targeting the existing vault"
   default     = "development"
   type        = string
 }
 
 variable "environment" {
   description = "The environment name to be used when creating AWS resources"
+  default     = "devops"
+  type        = string
+}
+
+variable "secrets_environment" {
```

Comment:
Not needed

Reply:
Implemented via commit: 252d107678658864551f07a8c8958fea666199bc